### PR TITLE
feat: Allow small text for unordered lists

### PR DESCRIPTION
**Pull request by alimpens**
Do we want the use the same API as for Paragraph? So `size="small"`? Or a `small` boolean?

### CODE REVIEW

**Comment by RubenSibon** [comment]
> Do we want the use the same API as for Paragraph? So `size="small"`? Or a `small` boolean?

Yes, keep it consistent for the developers.

**Review by RubenSibon — APPROVED** [review]

File: packages/react/src/UnorderedList/UnorderedList.tsx [hunk 1]
```diff
@@ -12,11 +12,13 @@ export type UnorderedListProps = {
   /** Changes the text color for readability on a dark background. */
   inverseColor?: boolean
   markers?: boolean
+  /** The size of the unordered list. */
```
Comment (RubenSibon) [inline packages/react/src/UnorderedList/UnorderedList.tsx:15]: What are our guidelines for documenting comments in code? Not every prop has comment, I see. When do we add one and when not? Why not JSDoc styled comments?

**Review by RubenSibon — APPROVED** [review]

File: packages/react/src/UnorderedList/UnorderedList.tsx (hunk 1 above)
Comment (alimpens) [reply packages/react/src/UnorderedList/UnorderedList.tsx:15]: We haven't done so in the past, but all props should have documentation. We have [two](https://gemeente-amsterdam.atlassian.net/jira/software/c/projects/DES/boards/1378?selectedIssue=DES-752) [tickets](https://gemeente-amsterdam.atlassian.net/jira/software/c/projects/DES/boards/1378?selectedIssue=DES-753) to fix that.

> Why not JSDoc styled comments?

I'm not quite sure what you mean? This is JSDoc.

Comment (RubenSibon) [reply packages/react/src/UnorderedList/UnorderedList.tsx:15]: Right, a different style than I'm used to (documenting props/params on the Function/Component level), but this is fine by me. Is it an idea to add comments for all props when we touch a file (i.e. Boy Scout Rule)?

Comment (alimpens) [reply packages/react/src/UnorderedList/UnorderedList.tsx:15]: The idea now is to add comments for all props in 2 batches (see the tickets I linked earlier). I think that's doable, there's a lot of repetition there.

**Pull request merged**